move Aardappel.Cube2 to publishers name dot3labs.*

Wolf1098 opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28188)

Wolf1098 committed:
move Aardappel.Cube2 to publishers name dot3labs.*

Wolf1098 commented:
@msftbot do not automerge

Wolf1098 commented:
expecting a clash between Aardappel.Cube2 and the new(actual) publisher folder

Wolf1098 commented:
Not too sure what is going on here, it worked on my end... maybe a depends opengl2 error....

ItzLevvie commented on manifests/d/dot3labs/Cube2/2020.11.29/dot3labs.Cube2.yaml:
```diff
@@ -0,0 +1,30 @@
+Description: 'Cube 2: Sauerbraten is a free multiplayer & singleplayer first person shooter, the successor of the Cube FPS. Much like the original Cube, the aim of this game is fun, old school deathmatch gameplay and also to allow map/geometry editing to be done cooperatively in-game.'
+PackageUrl: http://sauerbraten.org/
+Installers:
+- Architecture: x86
```
```suggestion
- Architecture: x64
```

Wolf1098 replied:
Its a 2 in 1 installer...Would this change reflect that? it has both the x64 and x32. I believe. I haven't deconstructed the exe myself recently.

Wolf1098 committed:
Update dot3labs.Cube2.yaml

trying UWP app like approach, using the same installer specified for the 2 arches

Wolf1098 commented:
seems I might have to use a multifile manifest
